src/opm/io/eclipse/EclOutput.cpp

```diff
@@ -409,6 +528,30 @@ std::string EclOutput::make_doub_string(double value) const
     }
 }
 
+std::string EclOutput::make_doub_string_ix(double value) const
+{
+    char buffer [21];
+    std::sprintf (buffer, "%19.13E", value);
+
+    if (value == 0.0) {
+        return " 0.0000000000000E+00";
+    } else {
+        if (std::isnan(value))
+            return "NAN";
+
+        if (std::isinf(value)) {
+            if (value > 0)
+                return "INF";
+            else
+                return "-INF";
+        }
+
+        std::string tmpstr(buffer);
```

Comment:
This can just be `return std::string{buffer};` - no need to create the temporary `tmpstr`